packagingrunner/docker-entrypoint.sh

```diff
@@ -8,7 +8,12 @@ operation="${1:-complete}"
 
 case "$operation" in
 build)
-  bundle exec rake build:all
+  shift
+  if [ "$#" -gt 0]; then
+    bundle exec rake build "$@"  # should be able to pass st2 or st2mistral as packages_list but can't figure this rake part.
```

Comment:
Cool, thanks. This wasn't obvious from reading the rakefiles. I'll close this PR.